docker/Dockerfile

```diff
@@ -21,50 +21,55 @@ FROM ubuntu:18.04 AS builder-install
 #
 SHELL ["/bin/bash", "-c"]
 
+# Certain Installers make 'installations' easier by having a nice front-end. While this is great when you have a manual install, this becomes an issue during automated installations.
+ENV DEBIAN_FRONTEND=noninteractive
+
 # Install build tools and dependencies
 #
 RUN apt-get update -q -q && \
  apt-get upgrade --yes --force-yes && \
  apt-get install --yes --force-yes \
-  automake \
-  autoconf \
   alien \
+  autoconf \
```

Comment:
Not a lot here was changed, I just sorted everything. The changes are adding `postgresql-10` and `libpq-dev`